test/functional/services/remote/webdriver.ts

```diff
@@ -50,6 +51,13 @@ const certValidation: string = process.env.NODE_TLS_REJECT_UNAUTHORIZED as strin
 const SECOND = 1000;
 const MINUTE = 60 * SECOND;
 const NO_QUEUE_COMMANDS = ['getLog', 'getStatus', 'newSession', 'quit'];
+const downloadDir = resolve(__dirname, '../../../../target/functional-tests/downloads');
```

Comment:
If we import `REPO_ROOT` from `@kbn/dev-utils` this could be a bit easier to read
```suggestion
const downloadDir = resolve(REPO_ROOT, 'target/functional-tests/downloads');
```